NewForce-Cohort-9/newforce-c9-tabloid-cli-TabloidCLI
# MainMenu / Posts Update

Description
This implements CLI logic to accommodate for Blogs. It creates PostRepository, PostManager, and completes logic for allowing users to view, add, edit, and delete post entries. 

Type of change


Testing Instructions
Start TabloidCLI.sln
Run program
From console menu, enter 4 to enter Post Menu
From the Post Menu:

User should be able to press 1 to view a List of posts.
User should be able to press 2 to enter Add a new post.
User should be able to press 3 to Remove a post.
User should be able to press 4 to Edit an existing post.
User should be able to press 5 to view Post Details.
User should be able to press 0 to return to the main menu.
.

Checklist:
